File: src/pathoperator.go
package pdfjet

// PathOperator contains constants for path operations
var PathOperator = struct {
	Stroke                        string // Stroke the path
	CloseAndStroke                string // Close and then stroke the path
	Fill                          string // Close and fill the path
	FillAndStroke                 string // Close, fill and then stroke the path
	FillUsingEvenOddRule          string // Like 'f' but using even odd rule
	FillUsingEvenOddRuleAndStroke string // Like 'b' but using even odd rule
}{
	Stroke:                        "S",
	CloseAndStroke:                "s",
	Fill:                          "f",
	FillAndStroke:                 "b",
	FillUsingEvenOddRule:          "f*",
	FillUsingEvenOddRuleAndStroke: "b*",
}

File: net/pdfjet/PathOperator.cs
namespace PDFjet.NET {
public static class PathOperator {
    public static readonly string Stroke = "S";                         // Stroke the path
    public static readonly string CloseAndStroke = "s";                 // Close and then stroke the path
    public static readonly string Fill = "f";                           // Close ant fill the path
    public static readonly string FillAndStroke = "b";                  // Close, fill and then stroke the path
    public static readonly string FillUsingEvenOddRule = "f*";          // Like 'f' but using even odd rule
    public static readonly string FillUsingEvenOddRuleAndStroke = "b*"; // Like 'b' but using even odd rule
}
}
